drivers/serial/uart_ite_it8xxx2.c

```diff
@@ -15,6 +15,7 @@
 #include <pm/policy.h>
 #include <soc.h>
 
+#define LOG_LEVEL CONFIG_UART_LOG_LEVEL
 #include <logging/log.h>
 LOG_MODULE_REGISTER(uart_ite_it8xxx2, LOG_LEVEL_ERR);
```

Comment:
Is it correct to leave LOG_LEVEL_ERR in place like this?

Reply:
Nope. Good catch